tests/pages/firefox/accounts.py

```diff
@@ -12,7 +12,7 @@ class FirefoxAccountsPage(FirefoxBasePage):
 
     URL_TEMPLATE = '/{locale}/firefox/accounts/{params}'
 
-    _firefox_monitor_button_locator = (By.ID, 'fxa-monitor-submit')
+    _firefox_monitor_button_locator = (By.CLASS_NAME, 'js-monitor-button')
```

Comment:
Like `querySelector` in JS, this only matches the first element found with the class name in the page, so it won't cover two buttons. 

If desirable, I'd suggest making two different selectors (`_firefox_monitor_primary_button_locator`, `_firefox_monitor_secondary_button_locator`), and then having the test assert that both are visible more explicitly.

Reply:
Thank you! I updated `page1.py` and `page2.py` as firefox/accounts only has one. 😄